### PR TITLE
added stylus linter

### LINKED ISSUE #1154: Add stylus linter
It seems rather easy to add a stylus linter for vue files: https://vue-loader.vuejs.org/guide/linting.html#stylelint
Should be helpful to maintain a consistent style and prevent errors like this: https://github.com/yunity/karrot-frontend/pull/1153

EDIT: the linter above is just a CSS linter, need to check if it makes sense to apply it.

### PR DESCRIPTION
**Pull request by sarco3t**
Closes #1154

## What does this PR do?
* adds stylus linter by stylelint-webpack
## Checklist

- [ ] added a test, or explain why one is not needed/possible...
- [ ] no unrelated changes
- [ ] asked someone for a code review
- [ ] joined #karrot-dev channel at https://slackin.yunity.org
- [ ] added an entry to CHANGELOG.md (description, pull request link, username(s))

### CODE REVIEW

File: .stylelintrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+ "processors": ["@mapbox/stylelint-processor-arbitrary-tags"],
```
[the remark this answers is not in the document]
Comment (sarco3t) [reply .stylelintrc]: It's required for vue single file components, `stylelint` doesn't work without this processor with them.

**Comment by sarco3t** [comment]
@tiltec I tried stylelint autofixing with different options and plugins, but i didnt get any success with it. `stylelint-config-recommended` rules pack doesn't work with stylus/sugarss parser, also.

**Comment by tiltec** [comment]
I played a bit around with styleint:
- it doesn't [check if properties have spelling mistakes](https://github.com/stylelint/stylelint/issues/935) -> wouldn't prevent #1153
- it's not a stylus linter actually, the SugarSS style seems mostly compatible but doesn't guarantee it -> might cause false positives
- it lints the commented style tag in `quasar.variables.styl`

So I'm not sure how much we gain from it really.

**Comment by sarco3t** [comment]
As I seen, postcss doesn't work with stylus now, and recommend to use
`sugarss` linter

**Comment by tiltec** [comment]
Given that https://github.com/SimenB/stylint has some activity again, I will close this PR and evaluate in future if we want to have a CSS linter at all.

Thank you for your initiative @sarco3t!

**Pull request closed without merging**